client-app/src/examples/portfolio/PortfolioPanelModel.js

```diff
@@ -62,7 +62,7 @@ export class PortfolioPanelModel {
             run: (position) => {
                 this.detailPanelModel.setPositionId(position ? position.id : null);
             },
-            delay: 500
+            debounce: 500
```

Comment:
This will be a debounce on top of the grid's built-in selection debounce. Do we really need it now that the grid debounce will be working as intended? Maybe loading all the position details is intensive enough that it's warranted still?

Reply:
To be clear, the grid has just a hyper minimal debouncing (0ms) between the agGrid and our selection model, just to bulk up multi-selection events that are generated by agGrid as streams of events.

This much larger debouncing here, is really aimed at keeping the master detail from going crazy with network requests.

Reply:
reducing it to 300 per our early convo -- improved debounce behavior does warrant that!